### PR TITLE
feat: env fix

### PR DESCRIPTION
**Pull request by SudoSurya**
By submitting this pull request, I confirm that my contribution is made under the terms of the MIT license.

## Problem/Feature

- removed example env files and binded env values to default env file directly so user can spin docker direclty without changing env files #140 
- separate makefile for none db projects and sqlite(sqlite doesnt have docker file adding docker commands in make file of sqlite project doesnt make sense )

## Checklist

- [x] I have self-reviewed the changes being requested
- [ ] I have updated the documentation (if applicable)

### CODE REVIEW

**Comment by SudoSurya** [comment]
i closed #145 and opened this pr (no new changes )

File: cmd/program/program.go [hunk 1]
```diff
@@ -317,11 +309,20 @@ func (p *Project) CreateMainFile() error {
...
-		return err
+	if p.DBDriver == "sqlite" || p.DBDriver == "none" {
+		// inject makefile template
+		makeFileTemplate := template.Must(template.New("makefile").Parse(string(framework.NonDbMakeFileTemplate())))
+		err = makeFileTemplate.Execute(makeFile, p)
+		if err != nil {
+			return err
+		}
+	} else {
+		// inject makefile template
```
Comment (Melkeydev) [inline cmd/program/program.go:320]: nit: Maybe change this to say this one includes the commands for Databases as well

**Review by Melkeydev — APPROVED** [review]
One small nit

**Comment by Melkeydev** [comment]
One thing I noticed is that if a user choose SQLite, they are kind of left thinking what should they next, since nothing is generated.

**Review by Melkeydev — APPROVED** [review]
small nit but up to you - LGTM

**Comment by SudoSurya** [comment]
> One thing I noticed is that if a user choose SQLite, they are kind of left thinking what should they next, since nothing is generated.

makefile will be generated for sqlite also but it doesn't not include any commands related to database (we have only docker commands in other databases since we don't have any docker commands for sqlite we generating normal makefile with doesnt include docker commands)

**Pull request merged**